src/ppx/instrument.ml

```diff
@@ -909,6 +911,10 @@ struct
     in
     let bisect_file = ast_convenience_str_opt !bisect_file in
     let bisect_silent = ast_convenience_str_opt !bisect_silent in
+    let bisect_sigterm =
+      let bool_s = (if !bisect_sigterm then "true" else "false") in
+      Exp.construct ~loc {txt = Longident.parse bool_s; loc} None
```

Comment:
```suggestion
      if !bisect_sigterm then [%expr true] else [%expr false]
```

Reply:
Agree with this, but it also needs `let open Parsetree in` on my machine.

Reply:
As here:

https://github.com/aantron/bisect_ppx/blob/c3b8b5cd6f74ecbcfb2b3a4ae2b172e3f1fd93a6/src/ppx/instrument.ml#L168-L175

Reply:
Fixed in c858d69.